indexing-service/src/main/java/org/apache/druid/indexing/common/actions/CleanMetadataAction.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.common.actions;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.apache.druid.indexing.common.task.Task;
+
+public class CleanMetadataAction implements TaskAction<Void>
```

Comment:
Please add a javadoc here.
I would prefer using some kind of response object rather than void. This response object could potentially contain the information of whether the cleanup was successful or not, how many rows of which table were deleted, etc. (probably not needed in this PR, but let's return something meaningful anyway.)

Also, rename:
```suggestion
public class CleanupMetadataAction implements TaskAction<Void>
```

Reply:
Done